FAQ.md

```diff
@@ -36,11 +44,27 @@ Run this code with the module installed (use your parameters within),
 and see the output:
 
 ```python
-python -c 'import logging, sys; logging.basicConfig(stream=sys.stderr, level=logging.DEBUG); from validate_email import validate_email; print(validate_email(\'your.email@address.com\', check_mx=True, debug=True))'
+python -c 'import logging, sys; logging.basicConfig(stream=sys.stderr, level=logging.DEBUG); from validate_email import validate_email; print(validate_email(\'your.email@address.com\', smtp_debug=True))'
 ```
 
 If you still don't understand why your code doesn't work as expected by
 looking at the the logs, then (and only then) add an issue explaining
 your problem with a REPRODUCIBLE example, and the output of your test
 run.
 
+## How can I pass my email account's credentials? How can I use port 465 or 587 when my provider blocks port 25?
+
+The credentials you got from your email provider, as well as the
+instruction to use port 465 or 587, refer to *your provider's* server
+for *outgoing* emails.
+
+This module, however, directly talks to the *recipient's* server for
+*incoming* emails, so neither your credentials nor the switch to port
+465 or 587 is of any use here.
+
+If your internet connection is within a dynamic range (often the case
+for private use) or it doesn't have a proper reverse DNS entry, the
+servers for many email domains will reject connections from you. This
```

Comment:
... servers for many email domains (depending on their configuration) will reject connections ...